libafl/src/corpus/cached.rs

```diff
@@ -115,6 +115,13 @@ where
         Ok(testcase)
     }
 
+    /// Removes an entry from the corpus, returning it if it was present; considers both enabled and disabled testcases.
+    fn remove_from_all(&mut self, idx: CorpusId) -> Result<Testcase<Self::Input>, Error> {
```

Comment:
Shouldn't `remove` just remove from wherever? Why does it make a difference if the testcase is disabled or not?

Reply:
every thing without `all` does not touch disabled entry

Reply:
IMHO remove should remove, no matter what

Reply:
Ok i'll change